.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Exclude files that don't need to be present in packages (so they're not downloaded by Composer)
+/.gitattributes  export-ignore
+/.gitignore      export-ignore
+/phpcs.xml       export-ignore
+/phpunit.xml     export-ignore
+/.github         export-ignore
+/tests           export-ignore
```

Comment:
unfortunately we cannot drop /tests as .... per very old design ... we use tests same abstract test case in the other packages - see:
https://github.com/php-mock/php-mock/blob/8f58972dce4de5a804dc0459383a11bc651416cf/tests/AbstractMockTestCase.php#L20
https://github.com/php-mock/php-mock-phpunit/blob/c1d0c2fbad486a9034a26fad170af7aa585b4e44/tests/PHPMockTest.php#L5